test/cppsuite/configs/search_near_01_default.txt

```diff
@@ -0,0 +1,21 @@
+# Configuration for search_near_01.
+# need to be defined.
+duration_seconds=60,
+cache_size_mb=1000,
+timestamp_manager=
+(
+    enabled=false,
+),
+workload_generator=
+(
+    populate_config=
```

Comment:
Would you be able to stress more with bigger values in the populate phase?

Reply:
The number of generated entries are actually quite big. The test uses this config to generates that number of entries for each prefix key. Currently the test has prefixes for aaa -> zzz. This means each prefix will generate 50 keys. I think it is abit misleading, however couldn't think of a way to make this better @etienneptl what are your thoughts?

Reply:
You can add this ^ as a comment, it makes a lot of sense!